lib/net/smtp.rb

```diff
@@ -831,6 +862,16 @@ def open_message_stream(from_addr, *to_addrs, &block)   # :yield: stream
 
     DEFAULT_AUTH_TYPE = :plain
 
+    # call-seq:
+    #   authenticate(username, secret, type = DEFAULT_AUTH_TYPE, **, &)
+    #
+    # Authenticates with the server, using the "AUTH" command.
+    #
+    # +type+ is the name of a SASL authentication mechanism.
+    #
+    # All arguments—other than +type+—are forwarded to the authenticator.
+    # Different authenticators may interpret the +username+ and +secret+
```

Comment:
Inaccurate for the current release.  It should be updated for this PR (and #72 can rename `user` to `username`).
```suggestion
    # Different authenticators may interpret the +user+ and +secret+
```